### DISCUSSION TITLE
contents/why-cors/

### DISCUSSION THREAD

**Discussion by giscus** [areumsheep/blog#23] [General]
# contents/why-cors/

브라우저의 과거와 함께 CORS 동작 과정에 대해 알아봅시다!

https://blog.areumsheep.vercel.app/contents/why-cors/

**Comment by BBAK-jun** [comment]
`너무너무` 잘 읽었습니다 감사합니다

**Reply by areumsheep** [reply]
도움이 되셨다니 다행이네요!
댓글 너무나 감사드립니다 :)
